<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetPaper</class>
 <widget class="QWidget" name="SetPaper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置试卷</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>541</width>
     <height>393</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_main">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QSplitter" name="splitter_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>试卷总分：</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_totalScore">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0分</string>
          </property>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>试卷总题数：</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_totalQuestionNum">
          <property name="text">
           <string>0题</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButton_openPaper">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>添加题型</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/resource/img/resource/img/choice.png</normaloff>:/resource/img/resource/img/choice.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_sendExamInfo">
          <property name="text">
           <string>发送试卷</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/resource/img/resource/img/sendPaper.png</normaloff>:/resource/img/resource/img/sendPaper.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>考试时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="hourComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>时</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="minComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>分</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="secComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>秒</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLineEdit" name="classNameLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="addClassNameButton">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>添加班级</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../新建文件夹/春木开发资料/basic1-171_add_new_plus-128.png</normaloff>../../新建文件夹/春木开发资料/basic1-171_add_new_plus-128.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="classNameComboBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>55</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="AWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>试卷预览 A</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="paperATextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>470</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
   <property name="plainText">
    <string>&lt;content&gt;&lt;/content&gt;





1.  lie
A. n.&amp;vi. 谎言; 说谎 v. 躺; 卧; 平放; 位于(lay, lain.
B. n. 蓝色 a.蓝色的；悲伤的，沮丧的
C. n. 骑自行车的人
D. n. 草药 答案

2.  appropriate
A. a. 有臭味的, 发出臭味的
B. a. 健康的, 适合的 v. （使）适合, 安装
C. ad.&amp;prep. 在……下面, 向……下面
D. a.恰当的

3.  mess
A. n. 冰岛
B. n. 宇宙飞船
C. n. 凌乱
D. n. 人群 vt. 拥挤, 群聚a reform．</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
